src/databricks/labs/ucx/queries/assessment/main/30_3_jobs.sql

```diff
@@ -1,11 +1,23 @@
-/* --title 'Incompatible Jobs' --width 6 */
+/*
+--title 'Incompatible Jobs'
+--width 6
+--overrides '{"spec":{
+    "encodings":{
+      "columns": [
+        {"fieldName": "job_name", "booleanValues": ["false", "true"], "linkUrlTemplate": "/jobs/{{ job_id }}", "linkTextTemplate": "{{ @ }}", "linkTitleTemplate": "{{ @ }}", "linkOpenInNewTab": true, "type": "string", "displayAs": "link", "title": "job_name"},
+        {"fieldName": "job_id", "booleanValues": ["false", "true"], "type": "integer", "displayAs": "number", "title": "job_id"},
```

Comment:
mark as hidden

Reply:
adding it to invisible columns did not hide it, removed it from the encoding. lmk if removing is not the right approach

Reply:
That is interesting, I would expect: remove it from encoding and add it to invisible columns